specviz/widgets/workspace.py

```diff
@@ -401,11 +401,13 @@ def generic_export(spectrum, path):
             data = {
                 'spectral_axis': spectrum.spectral_axis,
                 'flux': spectrum.flux,
-                'uncertainty': spectrum.uncertainty.array * spectrum.uncertainty.unit,
                 'mask': spectrum.mask if spectrum.mask is not None
```

Comment:
Do we really need to add `mask` here if it is `None`, or can we do the same thing we're doing with `uncertainty` below and only add it to the dict if it exists?